hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/StorageContainerManager.java

```diff
@@ -993,6 +995,10 @@ public void stop() {
       metrics.unRegister();
     }
 
+    if (ms != null) {
```

Comment:
in order of precedence, i think closing scmMetadataStore takes more precedence than Metrics. Shall we move this close statement to end.